framework/src/meshmodifiers/ElementDeleterBase.C

```diff
@@ -56,7 +56,28 @@ ElementDeleterBase::modify()
    *       but because the order of deletion might impact what happens to any existing sidesets or nodesets.
    */
   for (const auto & elem : deleteable_elems)
+  {
+    // On distributed meshes, we'll need neighbor links to be useable
+    // shortly, so we can't just leave dangling pointers.
+    //
+    // FIXME - this could be made AMR-aware and refactored into
```

Comment:
Do you really want to put this FIXME comment, here. Shouldn't you just put on issue on libMesh to add this feature.